### ISSUE TITLE
A11y: Investigate Interaction between VoiceOver and Experience WebViews (particularly with Formstack)

### ISSUE THREAD

**Issue by orospakr** [RoverPlatform/rover-ios#457]
Hypotheses:

a) iOS WebViews don't work with VoiceOver in general;

b) Formstack is not a11y compatible;

c) Rover has a bug interacting with formstack/webviews/voiceover preventing it from working correctly.

d) Something specific about the reporting customer's experience is causing the failure, but other cases are fine.

If c), then fix it.

**Comment by orospakr** [comment]
It appears to work fine out of the box, including in the customer's app.  Closing this for now.
